packages/components/src/public/SchemaQuery.ts

```diff
@@ -85,6 +85,11 @@ export class SchemaQuery {
         return false;
     }
 
+    hasSchemaQuery(sq: SchemaQuery): boolean {
```

Comment:
this looks like a near duplicate of `isEqual`. thoughts on just adding the `includeViewName` as an optional param to `isEquals` to replace this?